### PR TITLE
Enabled members lab setting for developer experiment flag

### LINKED ISSUE #10114: Create labs setting for members
This should enable/disable the use of the members auth middleware

### PR DESCRIPTION
**Pull request by rshbhgrg**
closes https://github.com/TryGhost/Ghost/issues/10114

- Members lab enabled to be always true behind developer flag

*Note*: This change uses hard-coded labs value for members based on `enableDeveloperExperiments` flag, ideal implementation for later is to pick those value from settings.

### CODE REVIEW

File: core/server/services/labs.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+const _ = require('lodash');
+const Promise = require('bluebird');
+const SafeString = require('./themes/engine').SafeString;
+const common = require('../lib/common');
+const config = require('../config');
+let labs = module.exports = {};
+
 
 labs.isSet = function isSet(flag) {
+    if (flag === 'members' && config.get('enableDeveloperExperiments')) {
```
Comment (allouis) [inline core/server/services/labs.js]: Is this gonna work for checking the labs flag in casper? e.g. https://github.com/TryGhost/Casper/blob/master/partials/site-nav.hbs#L23

Comment (rishabhgrg) [reply core/server/services/labs.js]: Not yet, have to check for that. 👍

File: core/server/services/themes/middleware.js [hunk 2]
```diff
@@ -60,6 +60,15 @@ themeMiddleware.updateTemplateData = function updateTemplateData(req, res, next)
         labsData = _.cloneDeep(settingsCache.get('labs')),
         themeData = {};
 
+    /**
+     * TODO: Uses hard-check for members prototype, removed here when added to settings
+    */
+    if (config.get('enableDeveloperExperiments')) {
```
Comment (allouis) [inline core/server/services/themes/middleware.js]: nice

File: core/server/services/labs.js (hunk 1 above)
Comment (allouis) [reply core/server/services/labs.js]: This should be an `||` right?

**Review by allouis — APPROVED** [review]

**Pull request merged**